ci/github-script/merge.js

```diff
@@ -10,13 +10,15 @@ function runChecklist({
   user,
   userIsMaintainer,
 }) {
-  const allByName = files.every(({ filename }) =>
-    filename.startsWith('pkgs/by-name/'),
+  const allByName = files.every(
+    ({ filename }) =>
+      filename.startsWith('pkgs/by-name/') && filename.split('/').length > 4,
```

Comment:
I think this better covered the intent of the constraints.

W.r.t. https://github.com/NixOS/nixpkgs/pull/457652#discussion_r2484751355: I still think talking about "files" may be clearer than "packages" as this more closely reflects what we're actually checking. The current wording _is_ accurate (now), but could be misinterpreted to be about package rebuilds instead of file diffs.

Also: I wonder if a regex better represents what we're trying to enforce here? Albeit, `/` isn't a great regex delimiter when dealing with paths: `/\/pkgs\/by-name\/..\/.+\/.+/`.

Reply:
> Does that make it better or worse?

Much clearer! Sorry we went round in circles so much over such a trivial detail